docs/backends.md

```diff
@@ -512,3 +512,43 @@ data:
   current-password-again: <path:secret-id#password#version-id>
   password-old: <path:secret-id#password#old-version-id>
 ```
+
+### 1Password Connect
```

Comment:
Does this backend support versioning? I don't see anything to suggest it does in the code, in which case we should add an explicit note here that specified versions will be ignored

Reply:
It does not. I will add a note